# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_redirect-caching_canceled.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit/test_redirect-caching_canceled.js
--- a/netwerk/test/unit/test_redirect-caching_canceled.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit/test_redirect-caching_canceled.js	Tue Feb 12 16:08:25 2019 +0000
@@ -37,7 +37,7 @@
 {
   Assert.equal(buffer, responseBody);
   var chan = make_channel(randomURI);
-  chan.asyncOpen2(new ChannelListener(secondTimeThrough, null));
+  chan.asyncOpen(new ChannelListener(secondTimeThrough, null));
 }
 
 function secondTimeThrough(request, buffer)
@@ -46,7 +46,7 @@
   var chan = make_channel(randomURI);
   chan.loadFlags |= Ci.nsIRequest.LOAD_FROM_CACHE;
   chan.notificationCallbacks = new ChannelEventSink(ES_ABORT_REDIRECT);
-  chan.asyncOpen2(new ChannelListener(finish_test, null, CL_EXPECT_FAILURE));
+  chan.asyncOpen(new ChannelListener(finish_test, null, CL_EXPECT_FAILURE));
 }
 
 function finish_test(request, buffer)
@@ -63,6 +63,6 @@
   httpServer.start(-1);
 
   var chan = make_channel(randomURI);
-  chan.asyncOpen2(new ChannelListener(firstTimeThrough, null));
+  chan.asyncOpen(new ChannelListener(firstTimeThrough, null));
   do_test_pending();
 }